src/api.json

```
@@ -65,6 +65,12 @@
   "light/get_definition": {
     "params": "string"
   },
+  "light/get_definition_chash": {
+    "params": {
+    "address": "string",
+    "max_mci?": "number"
```

The key max_mci should not have `?`, the type are defined in the file `index.d.ts`
```suggestion
    "max_mci": "number"
```